src/main/java/org/sqlite/util/OSInfo.java

```diff
@@ -162,6 +162,9 @@ else if (osName.contains("Linux")) {
         else if (osName.contains("AIX")) {
             return "AIX";
         }
+        else if (osName.contains("Android")) {
```

Comment:
@psini Did you check the actual value of `os.name` JVM property under Android?
I think it's just Linux, so this code wouldn't work as expected